src/analytics/Averages.js

```diff
@@ -18,7 +18,20 @@ const Averages = ({ cards, characteristics }) => {
 
   const counts = Object.entries(groups)
     .map((tuple) => {
-      const vals = tuple[1].map((card) => characteristics[characteristic](card)).filter((x) => x);
+      const vals = tuple[1]
+        .filter((card) => {
+          if (characteristic === 'CMC') {
```

Comment:
can you add a comment describing why we're accommodating this edge case? I don't want people to look at this in the future and get confused why we're doing this

Reply:
Sure